### DISCUSSION TITLE
Email received

### DISCUSSION THREAD

**Discussion by WahyuAgungBudiyanto** [simple-login/app#1621] [Q&A]
sir why takes to received catchallmail is between 5-10 minutes?

**Comment by nguyenkims** [comment]
Sometimes email providers can delay the reception of the email, this technics is called greylisting that you can read more about on https://en.wikipedia.org/wiki/Greylisting_(email)

On SimpleLogin side, a catch-all alias is immediately created and emails are immediately forwarded.
